### ISSUE TITLE
[vite-node] [ERR_MODULE_NOT_FOUND] after upgrade to Nuxt 3.0.0-rc.10

### ISSUE THREAD

**Issue by hussamBashat** [nuxt/framework#7647]
Hello everyone. Recently I deleted a package from the package.json because I do not need it. This led to the update of the other package and many problems appeared, so I had to upgrade to Nuxt 3.0.0-rc.10
But I couldn't find a solution to this problem. Any help? Thank you very much.

![Vite Error 500](https://user-images.githubusercontent.com/56697609/191014213-0b06f28e-d0f9-41ba-b321-4f288a5742c6.png)

**Comment by pi0** [comment]
Hi. This should be fixed in [edge channel](https://v3.nuxtjs.org/getting-started/upgrade-guide/edge-channel) with better trace of error. Can you please try it?

**Comment by hussamBashat** [comment]
> 

Hi, how can use edge channel?
Shall I start from remove lockfile?

**Comment by pi0** [comment]
Removing lockfile would be nice. It is in link of my comment above.

**Comment by lucassonsinlima** [comment]
I've experienced the same and this issue https://github.com/nuxt/framework/issues/7534#issuecomment-1248596609 led me to understand what's going on

**Comment by hussamBashat** [comment]
> 

Thank you it really helped me..

The `devBundler: 'legacy'` solved the **[vite-node notfound]** problem and also helped reveal the real problem with _'@heroicons'_ pachage.
I deleted the _'@heroicons'_package and Fate is running on client without `devBundler: 'legacy'` correctly.

But also the _'@headlessui'_ package was causing problems when refreshing the page so I had to add `devBundler: 'legacy' `anyway so that the server was working properly.
